core/src/env2/env_access/mutable.rs

```diff
@@ -333,6 +333,7 @@ where
     where
         R: scale::Decode,
     {
+        self.println("mutable::get_runtime_storage");
         T::get_runtime_storage(&mut self.buffer, key)
```

Comment:
Can you try printing the returned underlying buffer?

Reply:
Fixed now